centreon/src/App/Upgrade/Infrastructure/Dbal/DbalModuleRepository.php

```
@@ -295,8 +292,17 @@ private function executeModulePhpFile(string $filePath): void
             return;
         }
 
-        $pearDB = DbalConnectionAdapter::createFromDbalConnection($this->configConnection, $this->connectionConfig);
-        $pearDBO = DbalConnectionAdapter::createFromDbalConnection($this->realtimeConnection, $this->connectionConfig);
+        // TODO: temporary bridge — replace with proper ConnectionInterface injection once module upgrade scripts are migrated
+        $pearDB = new \CentreonDB(connectionConfig: $this->connectionConfig);
+        $pearDBO = new \CentreonDB(connectionConfig: new ConnectionConfig(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$pearDBO'. (undefined)

(UnusedLocalVariable)